core/templates/dev/head/pages/review-test-page/review-test-page.directive.ts

```diff
@@ -57,10 +57,18 @@ oppia.directive('reviewTestPage', ['UrlInterpolationService', function(
               story_id: ctrl.storyId
             });
           $http.get(reviewTestsDataUrl).then(function(result) {
+            var skillList = [];
+            var skillDescriptions = [];
+            for (var skillId in result.data.skills_with_description) {
+              skillList.push(skillId);
+              skillDescriptions.push(
```

Comment:
This is skillsWithDescriptions and not skillDescriptions right? Can we rename it if so?

Reply:
No it actually holds the skill Descriptions keyed by the skill id.

Reply:
I left a note in the corresponding backend code.